scripts/build/toolchain/termux_setup_toolchain_26.sh

```diff
@@ -171,8 +171,8 @@ termux_setup_toolchain_25c() {
 			$_TERMUX_TOOLCHAIN_TMPDIR/bin/$HOST_PLAT-clang.no-16-porting
 		cp $_TERMUX_TOOLCHAIN_TMPDIR/bin/$HOST_PLAT-clang.no-16-porting \
 			$_TERMUX_TOOLCHAIN_TMPDIR/bin/$HOST_PLAT-clang.16-porting
-		sed -i 's/"\$@"/--start-no-unused-arguments -Werror=implicit-function-declaration -Werror=implicit-int -Werror=int-conversion -Werror=incompatible-function-pointer-types --end-no-unused-arguments \0/g' \
```

Comment:
Why was this changed? Are we changing `-Werror` to `Wno-error` here? Might that cause us to silently miscompile code instead of detecting it at compilation time?

Reply:
Reading up a bit, I see the `TERMUX_PKG_ENABLE_CLANG16_PORTING` flag now.

So the plan is to gradually port over packages to build with more strict clang flags over time?

But shouldn't wee keep the strict flags in `-clang.16-porting`?

Reply:
Sorry, I was confused, I think I understand this now!